tests/stage/test_manager.py

```diff
@@ -0,0 +1,23 @@
+from snowcli.cli.stage.manager import StageManager
+
+
+def test_quote_stage_name():
```

Comment:
This works, potential improvement: using parameterized test. It will require importing unittest and pytest. 
This will allow you to define the input and output in the parameterization decorator, and prevent having to type out `StageManager.quote_stage_name` multiple times. Therefore less maintenance in the future if the method signature changes.